twitch4j-ktx/build.gradle.kts

```
@@ -0,0 +1,29 @@
+// In this section you declare the dependencies for your production and test code
+dependencies {
+	// Twitch4J Modules
+	// We use compileOnly so using this library does not require all modules.
+	// This does mean that using code that references modules not available will crash.
+	// This shouldn't be an issue as most, if not all, code are extension functions.
+	compileOnly(project(":twitch4j"))
+
+	// Kotlin coroutines
+	api(group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core")
```

adding `stdlib-jdk8` because we are supporting JDK8+. Gradle plugin will import only `stdlib`
```suggestion
  api(kotlin("stdlib-jdk8"))
  api(group = "org.jetbrains.kotlinx", name = "kotlinx-coroutines-core")
```

---

We're not using anything from the jdk8 kotlin lib, so this unnecessary

---

so there is no reason to set `kotlinOptions.jvmTarget` in `KotlinCompile` tasks

---

don't need to explicitly add `stdlib`: https://kotlinlang.org/docs/gradle.html#dependency-on-the-standard-library

we *could* specify `jvmTarget` (but it already defaults to `1.8`): https://kotlinlang.org/docs/gradle.html#attributes-specific-to-jvm